charts/nri-bundle/requirements.yaml

```diff
@@ -7,7 +7,7 @@ dependencies:
   - name: nri-prometheus
     repository: file://../nri-prometheus
     condition: prometheus.enabled
-    version: 1.8.2
+    version: 1.9.0
 
   - name: nri-metadata-injection
     repository: file://../nri-metadata-injection
```

Comment:
We could disable as well the prometheus scrape label in the KSM subchart in case of lowDataMode=true? @roobre

Reply:
https://github.com/newrelic/helm-charts/pull/376